src/test/java/com/hyperlocal/server/ShopControllerTest.java

```diff
@@ -278,7 +278,7 @@ public void shouldReturnArrayOfShops() throws Exception {
 
     /* ACT */
 
```

Comment:
There is no test for public CompletableFuture<List<SearchSnippet>> getDataFromSearchIndex

Reply:
Added a test for that